src/components/screens/multiSignature/editor/editor.test.js

```diff
@@ -1,7 +1,38 @@
 import React from 'react';
 import { mount } from 'enzyme';
+
+import { getTransactionBaseFees, getTransactionFee } from '@api/transaction';
+import useTransactionFeeCalculation from '@shared/transactionPriority/useTransactionFeeCalculation';
+import { fromRawLsk } from '@utils/lsk';
+
 import Editor from './editor';
 
+jest.mock('@shared/transactionPriority/useTransactionFeeCalculation');
```

Comment:
Now that node does not warn and instead fails with `process.exit(~0)` on unhandledPromiseRejections, our test setups have to be better. This fee calculation mock was missing.